recipes/super_state_machine/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.0" %}
+
+package:
+  name: super_state_machine
+  version: {{ version }}
+
+source:
+  url: https://github.com/beregond/super_state_machine/archive/{{ version }}.tar.gz
+  fn: suitcase-{{ version }}.tar.gz
+  sha256: a4aac93307f381c64e25b24de347b0527608cc551b66f41c34b2f2df80b445b3
+  patches:
+    - nuke_enum34.patch  # [py3k]
```

Comment:
@ocefpaf Neat!  Thanks for pointing that out. I definitely did not realize that was the case.  I can get rid of the patch